test/mux.sh

```diff
@@ -282,8 +299,8 @@ cleanup() {
     fi
     # delete indices created by this test
     curl_es $es_pod /project.testproj.* -XDELETE
-    curl_es $es_pod /project.default.* -XDELETE
-    curl_es $es_pod /project.mux-undefined.* -XDELETE
+    os::log::debug $( curl_es $es_pod /project.default.* -XDELETE ) || :
```

Comment:
This will always fail now - there should never be "project.default.*" indices

Reply:
Ok. Removing the line.

Reply:
@richm, it seems if we don't delete project.default.* here, it makes zzz-correct-index-names.sh fail later...
```
Running test/zzz-correct-index-names.sh:29: executing 'curl_es logging-es-data-master-jp217vjp-1-983bd /_cat/indices' expecting success and not text 'project.default.'...
FAILURE after 0.484s: test/zzz-correct-index-names.sh:29: executing 'curl_es logging-es-data-master-jp217vjp-1-983bd /_cat/indices' expecting success and not text 'project.default.': the output content test failed
```

Reply:
That means there is yet another bug somewhere - possibly we need to change something about the fluentd forward configuration so that default namespace messages go to .operations?  Otherwise, it is a bug.